src/commands/dev/dev.mjs

```diff
@@ -247,10 +257,25 @@ export const createDevCommand = (program) => {
         .hideHelp(),
     )
     .addOption(new Option('--graph', 'enable Netlify Graph support').hideHelp())
-    .addOption(new Option('--sessionId [sessionId]', '(Graph) connect to cloud session with ID [sessionId]'))
+    .addOption(
+      new Option(
+        '--sessionId [sessionId]',
+        'Old, prefer --session-id. (Graph) connect to cloud session with ID [sessionId]',
+      ).hideHelp(true),
+    )
+    .option('--session-id [sessionId]', '(Graph) connect to cloud session with ID [sessionId]')
     .addOption(
       new Option(
         '-e, --edgeInspect [address]',
+        'Old, prefer --edge-inspect. Enable the V8 Inspector Protocol for Edge Functions, with an optional address in the host:port format',
+      )
+        .conflicts('edgeInspectBrk')
+        .argParser(validateShortFlagArgs)
+        .hideHelp(true),
+    )
+    .addOption(
+      new Option(
+        '-e, --edge-inspect [address]',
         'enable the V8 Inspector Protocol for Edge Functions, with an optional address in the host:port format',
       )
         .conflicts('edgeInspectBrk')
```

Comment:
I think this needs to change?

```suggestion
        .conflicts('edge-inspect-brk')
```

Reply:
😅 one would think! But I tested it. This seems to work the same as the options. We need to pass the camelCased option. Example:

I created:
```
  .addOption(new Option('--json', 'Output return values as JSON').hideHelp(true).conflicts('test-karin'))
  .addOption(new Option('--test-karin', 'bleep bloop'))
```

When used like this, I don't get a conflict warning:
`☁  testjes [main] ⚡  ../../cli/bin/run.mjs dev --json --test-karin`

Changed it to camelcase:
```
  .addOption(new Option('--json', 'Output return values as JSON').hideHelp(true).conflicts('testKarin'))
  .addOption(new Option('--test-karin', 'bleep bloop'))
```

Then I got a warning:
```
☁  testjes [main] ⚡  ../../cli/bin/run.mjs dev --json --test-karin
 ›   Error: option '--json' cannot be used with option '--test-karin'
 ›   See more help with --help
```

Reply:
And others kind of made the same assumption: https://github.com/tj/commander.js/issues/1796

Reply:
Interesting!